pymbolic/primitives.py

```diff
@@ -899,8 +899,8 @@ def {cls.__name__}_hash(self):
         def {cls.__name__}_init_arg_names(self):
             depr_key = (type(self), "init_arg_names")
             if depr_key not in self._deprecation_warnings_issued:
-                warn("__getinitargs__ is deprecated and will be removed in 2025. "
-                        "Use dataclasses.fields instead.",
+                warn("Attribute 'init_arg_names' of {cls} is deprecated and will "
+                        "be removed in 2025. Use dataclasses.fields instead.",
```

Comment:
The phrasing here is a bit odd, since it claims we'll magically remove these attributes in 2025. Reword?

Reply:
Tried to reword it a bit in https://github.com/inducer/pymbolic/compare/9a62b35b5063d40cdbbb2084414e124141f3406b..7eba5378d83bfd5d6146ed9a94f16478758a801a. Let me know if that sounds clearer!